jmix-flowui/flowui-test-assist/src/main/java/io/jmix/flowui/testassist/FlowuiTestAssistConfiguration.java

```diff
@@ -30,10 +30,13 @@
 import io.jmix.flowui.view.navigation.*;
 import org.springframework.beans.factory.config.BeanDefinition;
 import org.springframework.context.annotation.*;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
 import org.springframework.core.annotation.Order;
 
 @Configuration
 @Import({VaadinScopesConfig.class})
```

Comment:
Add `ComponentScan` annotation. And change all bean prefixes to `flowui` excluding existing beans.